src/plugins/discover/public/services/doc_views/components/doc_viewer_source/source.test.tsx

```diff
@@ -111,7 +111,7 @@ describe('Source Viewer component', () => {
         />
       </KibanaContextProvider>
     );
-    expect(comp.children()).toMatchSnapshot();
+    expect(comp.children().render()).toMatchSnapshot();
```

Comment:
Data Discovery would like to revert changes in `src/plugins/discover/public/services/doc_views/components/doc_viewer_source/source.test.tsx` and `source.test.tsx.snap` otherwise the functionality which tests are checking is excluded in snapshots.

Reply:
Thanks!